app/uk/gov/hmrc/merchandiseinbaggagefrontend/views/CheckYourAnswersPage.scala.html

```diff
@@ -43,18 +43,47 @@ <h2 class="govuk-heading-m" id="main-message">@messages("checkYourAnswers.detail
     <div class="govuk-summary-list__row">
      <dt id="@{s"categoryLabel_$index"}"  class="govuk-summary-list__key">@messages("checkYourAnswers.detailsOfTheGoods.category")</dt>
      <dd id="@{s"category_$index"}" class="govuk-summary-list__value" style="text-transform: capitalize">@goods.categoryQuantityOfGoods.category</dd>
+     <dd class="govuk-summary-list__actions">
+       <a id="@{s"categoryChangeLink_$index"}" href=@{s"${GoodsTypeQuantityController.onPageLoad(index + 1)}"}>
+        Change<span class="govuk-visually-hidden">name</span>
+       </a>
+     </dd>
     </div>
     <div class="govuk-summary-list__row">
      <dt id="@{s"quantityLabel_$index"}" class="govuk-summary-list__key">@messages("checkYourAnswers.detailsOfTheGoods.quantity")</dt>
      <dd id="@{s"quantity_$index"}" class="govuk-summary-list__value" style="text-transform: capitalize">@goods.categoryQuantityOfGoods.quantity</dd>
+     <dd class="govuk-summary-list__actions">
+      <a id="@{s"quantityChangeLink_$index"}" href=@{s"${GoodsTypeQuantityController.onPageLoad(index + 1)}"}>
+        Change<span class="govuk-visually-hidden">name</span>
+       </a>
+     </dd>
+    </div>
+    <div class="govuk-summary-list__row">
+     <dt id="@{s"vatRateLabel_$index"}" class="govuk-summary-list__key">@messages("checkYourAnswers.detailsOfTheGoods.vatRate")</dt>
+     <dd id="@{s"vatRate_$index"}" class="govuk-summary-list__value" style="text-transform: capitalize">@{s"${goods.goodsVatRate.value}%"}</dd>
+     <dd class="govuk-summary-list__actions">
+      <a id="@{s"vatRateChangeLink_$index"}" href=@{s"${GoodsVatRateController.onPageLoad(index + 1)}"}>
+        Change<span class="govuk-visually-hidden">name</span>
```

Comment:
should these not be @messages("site.change") rather than just a hardcoded english Change

Reply:
100%